test/stat_tracker_test.rb

```diff
@@ -137,43 +137,6 @@ def test_winningest_team
   end
 
 
-  #
-  # def test_team_with_most_tackles_in_a_season
-  #   assert_equal "FC Dallas", @stat_tracker.most_tackles("20122013")
-  #
-  # end
-  #
-  # def test_team_with_fewest_tackles_in_a_season
-  #   assert_equal "FC Cincinnati", @stat_tracker.fewest_tackles("20152016")
-  # end
-  #
-  # def test_most_accurate_team
-  #   assert_equal "FC Dallas", @stat_tracker.most_accurate_team("20122013")
-  # end
-  #
-  # def test_least_accurate_team
-  #   assert_equal "Houston Dynamo", @stat_tracker.least_accurate_team("20132014")
-  # end
-  #
-  # def test_it_can_return_the_worst_coach_of_the_season
-  #   # mike said skip, for now
-  #   skip
-  #   assert_equal "Alain Vigneault", @stat_tracker.worst_coach(20132014)
-  # end
-  #
-
-
-
-  ######  Move these tests somewhere else
-
-  # def test_it_can_return_total_games_by_season
-  #   expected = {
-  #     "20122013"=>5,
-  #     "20152016"=>4
-  #     }
-  #   assert_equal expected, @stat_tracker.total_games_by_season
-  # end
-  #
   def test_it_can_count_total_games_by_team
```

Comment:
Perhaps we can format this differently.

Reply:
Potentially, you're referring to the hash bracket?